includes/application/forwardApprover.php

```diff
@@ -81,7 +81,7 @@
 		
 		$User = findPending($status);
 		
-		if( $User == null )
+		if( $User == '' )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after IF keyword; 0 found
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 9

<!-- Comment by [SideCI](https://sideci.com) -->